infrastructure/platform/data-collector.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: datacollector
+spec:
+  template:
+    spec:
+      containers:
+        - name: data-collector
+          image: pocketsizefund/datacollector:0.1.0
+          ports:
+            - containerPort: 8080
+          resources:
+            requests:
+              cpu: 100m
+              memory: 64Mi
+            limits:
+              cpu: 100m
+              memory: 128Mi
+          env:
+            - name: ALPACA_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: platform
+                  key: alpaca_api_key
+            - name: ALPACA_API_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: platform
+                  key: alpaca_api_secret
+            - name: AWS_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: platform
+                  key: aws_access_key_id
+            - name: AWS_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: platform
+                  key: aws_secret_access_key
+            - name: S3_DATA_BUCKET_NAME
+              valueFrom:
+                secretKeyRef:
+                  name: platform
+                  key: s3_data_bucket_name
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: morning-trigger
+spec:
+  schedule: "0 18 * * 1-5"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
```

Comment:
for the data collector do we care if there are duplicate runs? in theory it would just overwrite with the same data? if that's the case an onFailure restartPolicy might be useful?